### DISCUSSION TITLE
Listing enumerations in the generated docs

### DISCUSSION THREAD

**Discussion by claudio-alvaro-wbgames** [mkdocstrings/mkdocstrings#633] [Q&A] [answered]
If my module contains:
```python
import enum

class MyEnum(enum.Enum):
    A = 0
    B = 1
    C = 2
```

The generated docs only list `class MyEnum` but none of its options listed. How do I customize the output for inheritors of a class like `enum.Enum` so all options are displayed in the generated docs?

Thanks

**Comment by pawamoy** [comment, marked as answer]
Each value is an attribute of the class, so you can either document them, or set `show_if_no_docstring: true` for the class.

```python
class MyEnum(enum.Enum):
    """Docs for the class."""
    A = 0
    """Docs for A."""
    B = 1
    """Docs for B."""
    C = 2
    """Docs for C."""
```

**Reply by claudio-alvaro-wbgames** [reply]
Awesome! Thanks for the quick response.
